src/Home/Home.js

```diff
@@ -38,7 +41,7 @@ class Home extends Component {
 
 function mapStateToProps(state){
     return{
-        isPageMaskShown: state.AppReducer.get('isPageMaskShown'),
+        isPageMaskShown: state.ModalReducer.get('isPageMaskShown'),
```

Comment:
Reminder -- you don't need to put `Reducer` in the name, just `state.modal` would be fine